packages/devtools_app/lib/src/timeline/flutter/flutter_frames_chart.dart

```diff
@@ -45,21 +46,31 @@ class FlutterFramesChart extends StatefulWidget {
 class _FlutterFramesChartState extends State<FlutterFramesChart>
     with AutoDisposeMixin
     implements OnChartValueSelectedListener {
-  TimelineController _controller;
+  static const maxFrames = 150;
+
+  /// Datapoint entry for each frame duration (UI/GPU) for stacked bars.
+  final _frameDurations = <BarEntry>[];
 
-  List<TimelineFrame> frames = [];
+  /// Set of all duration information (the data, colors, etc).
+  BarDataSet frameDurationsSet;
 
   BarChartController _chartController;
 
   BarChartController get chartController => _chartController;
 
-  /// Datapoint entry for each frame duration (UI/GPU) for stacked bars.
-  final List<BarEntry> _frameDurations = <BarEntry>[];
+  TimelineController _controller;
 
-  /// Set of all duration information (the data, colors, etc).
-  BarDataSet frameDurationsSet;
+  int indexOffset = 0;
+
+  final lightTypeFace = TypeFace(
```

Comment:
make these const instead of final and move them outside the class into a file with other styles for devtools

Reply:
they can't be const because of FontWeight. moved them to the theme file.